controlplane/kubeadm/internal/controllers/scale_test.go

```diff
@@ -23,7 +23,10 @@ import (
 	"time"
```

Comment:
Q: Is it possible / is the plan to move the changes in scale_test.go, upgrade_test.go and the unrelated ones in controller_test.go in a separate PR?

This PR is already big and I would like to avoid having those changes here if they are not strictly necessary

(I'm struggling to find the time to review all of this in a timely manner)

Reply:
The changes in these files are necessary. The tests would fails otherwise, because the underlying code is now adopted to SSA. 
TLDR; all these tests eventually end up calling `createMachine` which is now using SSA and therefore would fail if we are not using SSA in the corresponding tests. All the other changes in made to the test are needed once we move to SSA.

Note: Setting `disableInPlacePropagation` to true would not help as `createMachine` is untouched by this flag.

Reply:
Alright, makes sense. Thx!